File: src/java/org/javaturk/dp/ch05/facade/computer/ROM.java
package org.javaturk.dp.ch05.facade.computer;

public class ROM {
	private int size;  // Size of ROM in MB
	
	public ROM(int size) {
		this.size = size;
	}

	public byte[] getOSKernelImage() {
		System.out.println("ROM: Loading the image of OS kernel.");
		byte[] kernelImage = new byte[2*1024*1024];
		
		// ...
		
		return kernelImage;
	}
}

File: src/typescript/com/gokselkucuksahin/dp/creational-patterns/builder/computer/domain/rom.ts
export class ROM {
  private size: number;

  constructor(size: number) {
    this.size = size; // Size of ROM in MB
  }

  getOSKernelImage(): number[] {
    console.log("ROM: Loading the image of OS kernel.");
    const kernelImage = Array(2 * 1024 * 1024);

    // ...

    return kernelImage;
  }
}